src/ontology/mondo-edit.obo

```diff
@@ -586047,6 +586047,16 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-03T14:16:10Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7719" xsd:anyURI
 
+[Term]
+id: MONDO:1040007
+name: hemihydranencephaly
+def: "A rare congenital brain disorder characterized by complete or almost near-complete unilateral absence of the cortical cortex, which is filled by a sac of cerebrospinal fluid. The presumed cause is a prenatal vascular disruption with occlusion of the carotid artery territories ipsilateral to the damaged brain." [https://orcid.org/0000-0001-9310-0163, https://orcid.org/0000-0002-0587-4693, PMID:23324549]
+synonym: "Hemi-hydranencephaly" EXACT []
```

Comment:
- a x-ref is needed for this synonym
- the synonym should not start with a capital letter

Note: I will update this.